linode/manager
# change: [M3-8520] - Revert the "View Code Snippets"button copy to the original text

## Description 📝
Revert the "View Code Snippets"button copy to the original text. This is a separate effort will be taken care with A/B testing ticket. 


## Target release date 🗓️

## How to test 🧪
(How to verify changes)
- Checkout the branch and toggle the  flag APICLI DX Tools in dev tools. 
- Verify the button copy.

## As an Author I have considered 🤔

*Check all that apply*
